patch-package
--- a/node_modules/@types/draft-js/index.d.ts
+++ b/node_modules/@types/draft-js/index.d.ts
@@ -950,6 +950,7 @@ import Editor = Draft.Component.Base.DraftEditor;
 import EditorProps = Draft.Component.Base.DraftEditorProps;
 import EditorBlock = Draft.Component.Components.DraftEditorBlock;
 import EditorState = Draft.Model.ImmutableData.EditorState;
+import EditorChangeType = Draft.Model.ImmutableData.EditorChangeType;
 
 import CompositeDecorator = Draft.Model.Decorators.CompositeDraftDecorator;
 import Entity = Draft.Model.Entity.DraftEntity;
@@ -999,6 +1000,7 @@ export {
     EditorProps,
     EditorBlock,
     EditorState,
+    EditorChangeType,
 
     CompositeDecorator,
     Entity,
